### ISSUE TITLE
how to explain the strange results

### ISSUE THREAD

**Issue by moshl** [mbourgey/EBI_cancer_workshop_CNV#1]
Dear Mathieu Bourge,
i am confused for my results: the _ratio has normal status but the _BAF has abnormal status (see attachement). Looking forwarding to your reply.
bestwishes,
moshl

[WY_3D30_gc50b_chromosome_view.pdf](https://github.com/mbourgey/EBI_cancer_workshop_CNV/files/1656080/WY_3D30_gc50b_chromosome_view.pdf)
[WY_3D30_gc50b_genome_view.pdf](https://github.com/mbourgey/EBI_cancer_workshop_CNV/files/1656082/WY_3D30_gc50b_genome_view.pdf)

**Comment by mbourgey** [comment]
I that case I would not trust the copy number value (20 copies). To my point of vew it looks like a LOH of the whole chromosome 13 (BAF almost 0 but normal depth).

**Comment by moshl** [comment]
so, if it is a LOH but normal depth, which does mean the genotype is AA? is the result credible?

**Comment by mbourgey** [comment]
yes for LOH you don't have a copy number change but all your SNP positions are homozygote. why the BAF is not fully 0 or 1, probably because of the cellularity so you should still have a few amount of normal cells which are not LOH for the region.
